### PR TITLE
partial fix for nighthawk rename

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

This PR is an attempt on #2317

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit dcd260d9a356a5c651a50279d7b794e1640580ed at: https://617f4ad38528ac48b995d360--layer5ng.netlify.app

File: README.md [hunk 1]
```diff
@@ -99,8 +99,8 @@ alt="Layer5" align="left" /></a>
 </p>
 <br />
 <p style="clear:both;">
-<h2><a href="https://layer5.io/smi">GetNightHawk </a></h2>
-<a href="https://getnighthawk.dev/"><img src=".github/assets/images/getnighthawk/layer5-getnighthawk.png"
+<h2><a href="https://layer5.io/smi">NightHawk</a></h2>
```
Comment (warunicorn19) [inline README.md:102]:
```suggestion
<h2><a href="https://layer5.io/smi">Nighthawk</a></h2>
```

**Review by warunicorn19 — APPROVED** [review]
LGTM, seems to cover all.

**Comment by l5io** [comment]
🚀 Preview for commit 20d16400c11b6984d175e1999f889b1ffdd79c88 at: https://617f969c6961c50e7a72bca8--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Thanks. It feels like I missed something. I think the brand page is using to same logo twice.

**Comment by warunicorn19** [comment]
Screenshot 2021-11-01 at 3 31 10 PM

seems to be fine right?

**Comment by leecalcote** [comment]
Regarding the seemingly duplicative entries for the logos, here is a screenshot from mobile view, which puts the otherwise separate text, below the icon making the logos look duplicated, but really aren't.

Screen Shot 2021-11-01 at 1 45 38 PM

@Mr-DG-Wick, you're right about C++ not directly being a selling point. We need to shift focus to highlight what types of performance insights are available to users and how they can be put into action by users based on the use of Nighthawk and not focus on implementation details.

**Comment by leecalcote** [comment]
It would be good to open a separate issue regarding hiding implementation details and highlighting value provided.

**Pull request merged**